android/app/src/main/res/layout/settings.xml

```diff
@@ -35,9 +57,9 @@
         android:layout_height="wrap_content" />
 
     <ListView
-		android:id="@+id/log_list"
-	    android:layout_width="match_parent"
-	    android:layout_height="wrap_content"
-	    android:paddingTop="10dip" />
+        android:id="@+id/log_list"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingTop="10dip" />
```

Comment:
Please watch out for (unrelated) whitespace changes next time. If you think whitespace (or coding style) should be changed, make this a separate commit. Cramming multiple things into one commit makes understanding the changes harder than necessary.